### PR TITLE
[LTD-2356] Clear assessments

### PR DESCRIPTION
**Pull request by alixedi**

### CODE REVIEW

File: caseworker/tau/views.py [hunk 1]
```diff
@@ -231,3 +231,35 @@ def post(self, request, queue_pk, pk):
...
+            payload = {
+                "control_list_entries": [],
+                "is_good_controlled": None,
+                "is_wassenaar": False,
+                "report_summary": None,
+                "comment": None,
+                "current_object": good["id"],
+                "objects": [good["good"]["id"]],
+            }
+            post_review_good(self.request, case_id=pk, data=payload)
```
Comment (saruniitr) [inline caseworker/tau/views.py:262]: if this raises an exception then we clear partially

File: caseworker/tau/views.py [hunk 2]
```diff
@@ -231,3 +231,35 @@ def post(self, request, queue_pk, pk):
...
+            **context,
+            "case": self.case,
+            "queue_id": self.queue_id,
+            "assessed_goods": self.assessed_goods,
+        }
+
+    def post(self, request, queue_pk, pk):
+        for good in self.assessed_goods:
+            payload = {
+                "control_list_entries": [],
```
Comment (saruniitr) [inline caseworker/tau/views.py:254]: should we replace it with the exporter provided clc here when clearing?

File: caseworker/tau/views.py (hunk 1 above)
Comment (alixedi) [reply caseworker/tau/views.py:262]: Yes, but I think this work will be followed by a v2 that will let users clear specific assessments by clicking checkboxes.

File: caseworker/tau/templates/tau/clear_assessments.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	{% include "tau/case_tabs.html" with case=case queue=queue %}
+{% endblock %}
+
+{% block full_width %}
+
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper tau">
+        <div class="govuk-grid-row">
+			<h1 class="govuk-heading-l">Clear all assessments</h1>
+			<p class="govuk-body">Are you sure you want to permanently delete all {{assessed_goods.count}} product assessments on this case?</p>
```
[the remark this answers is not in the document]
Comment (saruniitr) [reply caseworker/tau/templates/tau/clear_assessments.html:15]: :+1:

File: caseworker/tau/views.py (hunk 1 above)
Comment (saruniitr) [reply caseworker/tau/views.py:262]: :+1:

File: caseworker/tau/views.py (hunk 2 above)
Comment (saruniitr) [reply caseworker/tau/views.py:254]: :+1:

**Review by saruniitr — APPROVED** [review]
with the tests and lint errors fixed this looks good to me.

**Pull request merged**